apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/FDADrugDataUtilityApp.java

```diff
@@ -111,11 +111,11 @@ private static void buildProductsResource(Path convertedNdcDataFile, Path workin
 
     // unzip FDA NDC file
     unzip(downloadedNdcZipFile, workingDir);
-    Path originalNdcDataFile = workingDir.resolve("product.txt");
+    Path originalNdcDataFile = workingDir.resolve("product.xls");
     if (!Files.isReadable(originalNdcDataFile))
-      originalNdcDataFile = workingDir.resolve("Product.txt");
+      originalNdcDataFile = workingDir.resolve("Product.xls");
     if (!Files.isReadable(originalNdcDataFile))
-      throw new IllegalStateException("Unable to locate product.txt in " + ndctextZipUrl);
+      throw new IllegalStateException("Unable to locate product.xls in " + ndctextZipUrl);
```

Comment:
I would suggest putting this fix in a separate PR. Also: if the file format changed, in addition to the _extension_, the code that parses+processes it will also require updates.